ui_framework/components/table/landing_page_table/landing_page_tool_bar_footer.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { KuiToolBarPagerSection, KuiToolBarFooterSection, KuiToolBarText } from '../../tool_bar';
+
+export function LandingPageToolBarFooter({ selectedItemsCount, pagerState, onPreviousPage, onNextPage }) {
+  function getSelectedItemsSection() {
+    if (selectedItemsCount === 0) return null;
+
+    return <KuiToolBarFooterSection>
+      <KuiToolBarText>{selectedItemsCount} selected</KuiToolBarText>
+    </KuiToolBarFooterSection>;
+  }
+
+  return <div className="kuiToolBarFooter">
+      { getSelectedItemsSection() }
```

Comment:
I was thinking the check for null should exist here, not internally, so there would still be a conditional here, unless you disagree? 

```
  return <div className="kuiToolBarFooter">
      {
        selectedItemsCount > 0
          ? <KuiSelectedItemsFooterSection selectedItemsCount={selectedItemsCount}/>
          : null
      }
      <KuiToolBarPagerSection
        pagerState={pagerState}
        onNextPage={onNextPage}
        onPreviousPage={onPreviousPage}/>
    </div>;
```